easybuild/easyconfigs/f/FSL/FSL-6.y.z_add_missing_sripts.patch

```diff
@@ -0,0 +1,195 @@
+diff -urN fsl/src/fslio.orig/imcp fsl/src/fslio/imcp
```

Comment:
Please add the author and description for this patch as well.

Reply:
This patch seems to simply add missing scripts. This is a [known issue](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation#imcp.2Fimglob.2Fimmv_errors_after_installation) with FSL installatoin, and I would suggest following the earlier easyconfigs approach, as suggested in the FSL wiki, e.g.

https://github.com/easybuilders/easybuild-easyconfigs/blob/8a78d293bc6461b021d456f2c23b90e16d3f771d/easybuild/easyconfigs/f/FSL/FSL-6.0.2-foss-2019a-Python-3.7.2.eb#L54-L59

instead of just re-adding the scripts.

Reply:
@crubb you are right, thanks for the feedback
@HPC-UniOldenburg could you update this PR removing the patch `FSL-6.y.z_add_missing_sripts.patch ` and adding the same `postinstallcmds` from `FSL-6.0.2-foss-2019a-Python-3.7.2.eb`?